meta-dts-distro/recipes-support/dasharo-ectool/dasharo-ectool_0.3.8.bb

```diff
@@ -1,13 +1,21 @@
 inherit cargo
 inherit pkgconfig
 
-SRC_URI += "gitsm://github.com/Dasharo/ec.git;protocol=https;nobranch=1"
-SRCREV = "2b2c17ac6e61f45e0fc1bcf6b907a8289f37bcc4"
-S = "${WORKDIR}/git"
-CARGO_SRC_DIR = "/tool/"
+SUMMARY = "System76 Open Source Embedded Controller"
+HOMEPAGE = "https://github.com/Dasharo/ec"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = " \
+    file://tool/LICENSE;md5=af209eac18ec76ed06fb2839a906b1ad \
+"
+
+DEPENDS += "hidapi"
 PV:append = ".AUTOINC+411fab9b7c"
+SRC_URI:append = " gitsm://github.com/Dasharo/ec.git;protocol=https;nobranch=1"
```

Comment:
Why append here? It is first time `SRC_URI` is being set.

Reply:
I'm not sure - the original recipe used the `+=` operator here. Maybe you're right that it's better to just use `=` here. d9dea8d9b86629b097f6d3bbd8f01aee7df2e817